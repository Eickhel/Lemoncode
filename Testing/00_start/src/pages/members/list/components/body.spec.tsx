import { shallow } from "enzyme";
import * as React from "react";
import { Body } from "./body";

describe("pages/members/list/components/body tests", () => {
  it("should render as expected when passing required properties", () => {
    // Arrange
    const props = {
      members: [
        {
          id: 14540103,
          name: "antonio06",
          avatarUrl: "https://avatars1.githubusercontent.com/u/14540103?v=4"
        },
        {
          id: 1457912,
          name: "brauliodiez",
          avatarUrl: "https://avatars1.githubusercontent.com/u/1457912?v=4"
        },
        {
          id: 13205689,
          name: "crsanti",
          avatarUrl: "https://avatars1.githubusercontent.com/u/13205689?v=4"
        }
      ]
    };
    // Act
    const component = shallow(<Body {...props} />);

    // Assert
    expect(component).toMatchSnapshot();
  });
});
