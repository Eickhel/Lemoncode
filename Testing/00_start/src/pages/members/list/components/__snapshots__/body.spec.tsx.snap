// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`pages/members/list/components/body tests should render as expected when passing required properties 1`] = `
<tbody>
  <Component
    key="14540103"
    member={
      Object {
        "avatarUrl": "https://avatars1.githubusercontent.com/u/14540103?v=4",
        "id": 14540103,
        "name": "antonio06",
      }
    }
  />
  <Component
    key="1457912"
    member={
      Object {
        "avatarUrl": "https://avatars1.githubusercontent.com/u/1457912?v=4",
        "id": 1457912,
        "name": "brauliodiez",
      }
    }
  />
  <Component
    key="13205689"
    member={
      Object {
        "avatarUrl": "https://avatars1.githubusercontent.com/u/13205689?v=4",
        "id": 13205689,
        "name": "crsanti",
      }
    }
  />
</tbody>
`;
